tests/istio_test.go

```diff
@@ -38,3 +37,41 @@ func TestIstioClusterNode(t *testing.T) {
 func TestIstioDestinationRuleNode(t *testing.T) {
 	testNodeCreationFromConfig(t, istio.Manager, "destinationrule", objName+"-destinationrule")
 }
+
+func TestBookInfoScenario(t *testing.T) {
+	testRunner(
+		t,
+		[]Cmd{
+			{"kubectl apply -f /usr/bin/istio-1.0.1/samples/bookinfo/networking/destination-rule-all.yaml", true},
```

Comment:
please use a github link to master rather than the locally installed file, this will decouple installed istio installation script from istio test and i am assuming the example (on master) is backwards compatible with older istio versions

Reply:
DONE

Reply:
DONE